### ISSUE TITLE
change versioning scheme to something more friendly to frequent changes

### ISSUE THREAD

**Issue by tesselode** [Ogmo-Editor-3/OgmoEditor3-CE#70]
Since it seems like we're making changes relatively quickly, we should probably be listing things as alpha or beta. At least, that would make me feel more free to submit changes.

**Comment by AustinEast** [comment]
I'm fine with setting up a better defined versioning scheme 👍 Did you have any specifics in mind? 

also, if we were to do something like that - It'd be great if we got the versioning scheme documented somewhere, preferably in the same space as the docs from #66.

**Comment by tesselode** [comment]
people seem to like semantic versioning!

**Comment by 01010111** [comment]
I think that's probably the best thing - moving forward I think releases should be 
3.X.Y
X - Feature Updates
Y - Bugfixes

**Comment by AustinEast** [comment]
^That's generally the approach we're taking atm.

**Comment by tesselode** [comment]
what about breaking changes?

**Comment by 01010111** [comment]
That's kind of weird because this tool is specifically branded as OGMO Editor 3 - I think we should try to avoid breaking changes as much as possible (or at least handle them as smartly as possible). If we get to a point where there are tons of breaking changes and a large feature set behind them I guess it may make sense to consider OGMO Editor 4 :)

**Comment by AustinEast** [comment]
Yeah, things get a little fuzzy because the prior versions of Ogmo exist and aren't going anywhere. but maybe we should consider separating the name of the editor from its version number. That way we can have OGMO Editor 3 eventually be at version 5.4.2, for example.

**Comment by tesselode** [comment]
Yeah. I understand not wanting to make breaking changes, but I'm already thinking about things like changing the anchor of an entity in the project editor not updating existing entities...adjusting that behavior would be a breaking change, I think.

I also think it's not too late to go back and change some version numbers. We're only at 3.1.1 and I doubt anyone's using this for an important project yet...
